### ISSUE TITLE
Directed wires.

### ISSUE THREAD

**Issue by Aminion** [Qmunity/BluePower#331]
For more compact circuits, add the directed wires, which will be conduct signal in different directions. Types of wire direction - T, L, I, and ⅂L, which will redirect signals from opposite sides to other two outputs.
By rotation of these wires, all cases of wiring will be covered. This replacement for a RedPower covers, in scope of wiring problems.

**Comment by amadornes** [comment]
So basically Null Cells that instead of having one wire to go over another allow two connections making a corner to connect to eachother?
XAX
AXB
XBX
Is something like that what you mean?

**Comment by Aminion** [comment]
Null Cells decide a particular case - crossed wires. My idea decide all cases (I guess so). Here is  blueprint.
![types](https://cloud.githubusercontent.com/assets/6048639/6990072/8a565b36-da6d-11e4-9033-f6ac4cf57b12.png)

**Comment by amadornes** [comment]
We could add wrench modes to the Null Cell to do that

**Comment by Quetzi** [comment]
you could accomplish this using integrated circuits

**Comment by Aminion** [comment]
>you could accomplish this using integrated circuits
I think it will overkill, to solve this problem through the use of integrated circuits. Adding mode to Null Cells will be optimal.

**Comment by amadornes** [comment]
I agree with Aminion. Also ICs aren't working in the current version (I'm going to work on them today) so it can't really be done right now.
